packages/fxa-auth-server/bin/key_server.js

```diff
@@ -4,7 +4,7 @@
 
 'use strict';
 
-const config = require('../config');
+const config = require('../config').default;
```

Comment:
Missed this because of https://github.com/mozilla/fxa/blob/main/.gitignore#L77-L78 😭